simulator/func_sim/t/unit_test.cpp

```diff
@@ -179,7 +186,7 @@ TEST_CASE( "Torture_Test: MIPS32 calls without kernel")
     CHECK( sim->get_exit_code() == 0);
 }
 
-static auto get_simulator_with_test( const std::string& isa, const std::string& test, bool enable_hooks)
+static auto get_simulator_with_test( const std::string& isa, const std::string& test, bool enable_hooks, bool enable_ehandler= false)
```

Comment:
We do not use default arguments: https://github.com/MIPT-ILab/mipt-mips/wiki/Cpp-guidelines#default-arguments

Do I understand correctly the intention is to not instantiate the kernel for unsupported ISAs?
Then it's better to name it "enable_mars" and control "kernel" polymorphism instead of calling/not calling the initialization.